server/middlewares/articlesMiddleware.js

```diff
@@ -31,38 +32,36 @@ export default class AriclesMiddleware {
       categoryId = '',
     } = req.body;
 
-    if (typeof title !== 'string') {
-      return Response.send(res, STATUS.BAD_REQUEST, [], 'title must be a string', false);
+
+    if (!title) {
+      return Response.send(res, STATUS.BAD_REQUEST, [], 'The title of an article cannot be empty', false);
     }
-    if (typeof body !== 'string') {
-      return Response.send(res, STATUS.BAD_REQUEST, [], 'body must be a string', false);
+    if (!isNaN(title)) {
+      return Response.send(res, STATUS.BAD_REQUEST, [], 'The title of an article cannot contain only numbers or spaces', false);
     }
-    if (typeof description !== 'string') {
-      return Response.send(res, STATUS.BAD_REQUEST, [], 'description must be a string', false);
+    if (!description) {
+      return Response.send(res, STATUS.BAD_REQUEST, [], 'Please provide a short description for this article', false);
     }
-
-    // eslint-disable-next-line no-restricted-globals
-    if (isNaN(categoryId)) {
-      return Response.send(
-        res, STATUS.BAD_REQUEST, {}, 'category type must be an integer', false,
-      );
+    if (!isNaN(description)) {
+      return Response.send(res, STATUS.BAD_REQUEST, [], 'The description of this article cannot contain only numbers or spaces', false);
     }
-
-    if (!title || !title.trim()) {
-      return Response.send(res, STATUS.BAD_REQUEST, [], 'title cannot be empty', false);
+    if (description.length > 255) {
+      return Response.send(res, STATUS.BAD_REQUEST, [], 'Description cannot be more than 255 characters', false);
     }
-    if (!body || !body.trim()) {
-      return Response.send(res, STATUS.BAD_REQUEST, [], 'body cannot be empty', false);
+    if (!body) {
+      return Response.send(res, STATUS.BAD_REQUEST, [], 'Please provide the details for this article', false);
     }
-    if (!description || !description.trim()) {
-      return Response.send(res, STATUS.BAD_REQUEST, [], 'description cannot be empty', false);
+    if (!isNaN(body)) {
```

Comment:
#### Nitpick
- Use regular expressions instead of `isNaN`.

Reply:
resolved